saferpayofficial.php

```diff
@@ -206,15 +206,29 @@ public function hookPaymentOptions($params)
             \Invertus\SaferPay\Service\PaymentRestrictionValidation::class
         );
 
+        $logosEnabled = $paymentRepository->getAllLogosEnabled();
```

Comment:
getAllActiveLogosNames ? will allow to understand that we retrieving names not Logos object or ids